flink-yarn-tests/src/test/java/org/apache/flink/yarn/YarnTestBase.java

```diff
@@ -163,7 +160,7 @@
 
 	protected static File yarnSiteXML = null;
 
-	private YarnClient yarnClient = null;
+	protected YarnClient yarnClient = null;
```

Comment:
We already have a getter for the `yarnClient` which we use in the `YARNITCase.testPerJobMode`.
The same `getYarnClient` can be used in `waitApplicationFinishedElseKillIt` then there is no need for this `protected` change.

Reply:
Done, I didn't see it.